src/pages/home/home.scss

```diff
@@ -164,4 +164,21 @@ p {
 		display: flex;
 		justify-content: center;
 	}
+	.donate-button {
+		font-size: 0.875em;
+		padding: 10px;
+		background: #7000ff;
+		color: white !important; 
+		letter-spacing: 2px;
+		text-transform: uppercase;
+		border-radius: 3px;
+		border: 2px solid #7000ff;
+		margin: 3rem auto;
+		display: center;
+	}
+	
+	.donate-button:hover {
+		color: white;
+		background: #7000ff;
+	}
```

Comment:
I imagine a donate button may be used in more than one location throughout the site. It should be a reusable component with it's own stylesheet. This scss should be refactored to there.